<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>面向物联网实时多目标跟踪应用的并行加速系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>19</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>视频信息</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>视频类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="videotypeLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="locationurlLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>路径/网址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="videonameLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>视频名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="resolutionLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>分辨率：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>180</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>处理信息</string>
    </property>
    <widget class="QLabel" name="trackingmethodLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>多目标跟踪算法：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>当前帧率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="fpsLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="framenumberLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Uncertain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>当前帧数：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>端边协同加速选项</string>
    </property>
    <widget class="QCheckBox" name="enableedgeChb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>92</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开启</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(32, 74, 135);
</string>
     </property>
     <property name="text">
      <string>网络环境采样数</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="KSpb">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>基础加速选项</string>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>时空间优化策略</string>
     </property>
     <widget class="QCheckBox" name="timetrickChb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>在时间维度上进行优化</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="spacetrickChb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>在空间维度上进行优化</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="detcycleSpb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="globaldetcycleSpb">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>48</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(164, 0, 0);
</string>
      </property>
      <property name="text">
       <string>检测周期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(92, 53, 102);

</string>
      </property>
      <property name="text">
       <string>全局检测周期</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>使用TensorRT加速推理框架</string>
     </property>
     <widget class="QCheckBox" name="enabletrtChb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>92</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开启</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="enablefp16Chb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>92</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>使用fp16</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>581</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>跟踪视频</string>
    </property>
    <widget class="QLabel" name="videoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>561</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>【视频待录入... ...】</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionContinue"/>
   <addaction name="actionPause"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开..</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normaloff>../pause.png</normaloff>../pause.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset>
     <normaloff>../continue.png</normaloff>../continue.png</iconset>
   </property>
   <property name="text">
    <string>continue</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>../close.png</normaloff>../close.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
